perllib/Open311/PopulateServiceList.pm

```diff
@@ -283,6 +285,21 @@ sub _set_contact_group {
     }
 }
 
+sub _set_contact_non_public {
+    my ($self, $contact) = @_;
+
+    # We never want to make a private category unprivate.
+    return if $contact->non_public;
+
+    my $keywords = $self->_current_service->{keywords} || '';
+    $contact->update({
```

Comment:
It seems a bit odd that one contact can cause multiple changelog entries (e.g. each of name, group, extra qns and private make their own row), but something for later perhaps.